kernel/src/syscall/sched_getattr.rs

```diff
@@ -47,6 +49,14 @@ pub(super) struct LinuxSchedAttr {
     pub(super) sched_util_max: u32,
 }
 
+// Reference: <https://elixir.bootlin.com/linux/v6.17.7/source/include/uapi/linux/sched/types.h#L7>
+const SCHED_ATTR_SIZE_VER0: u32 = 48;
+// Reference: <https://elixir.bootlin.com/linux/v6.17.7/source/include/uapi/linux/sched/types.h#L8>
+#[cfg_attr(target_arch = "x86_64", expect(dead_code))]
+const SCHED_ATTR_SIZE_VER1: u32 = 56;
```

Comment:
Why is `SCHED_ATTR_SIZE_VER1` dead code only on x86_64? It looks like other architectures don’t reference it either.

Reply:
I have the same question. But I guess it's because:
https://github.com/asterinas/asterinas/blob/121da78aa2576da02294662ee1ca279aa1316c0f/kernel/src/syscall/mod.rs#L5-L8